kuma/static/styles/components/structure/archive.scss

```diff
@@ -1,15 +1,9 @@
-@import 'includes/vars';
-
 /*
-Archived pages
+For articles that are archive content. Makes them stand out more.
 ********************************************************************** */
 
-/*
-Red background & stripped border for archived pages
-- intentionally odd
-********************************************************************** */
 
-.wiki-main-content {
+.archive-content {
     background-color: $red-light;
     border: 10px solid $red;
     border-image: repeating-linear-gradient(-45deg, $red 0, $red 5px, $yellow 5px, $yellow 10px, $red 10px) 10 10 repeat;
```

Comment:
~~I would add a bottom margin of 20px to add some space between the content block and the article metadata.~~

Before

![Screenshot 2019-07-19 at 08 47 35](https://user-images.githubusercontent.com/10350960/61515082-fa261600-aa01-11e9-802f-e4db7cc95244.png)

After

![Screenshot 2019-07-19 at 08 47 42](https://user-images.githubusercontent.com/10350960/61515088-001bf700-aa02-11e9-8ffd-b9dc0c80bdf2.png)

Reply:
~~To get the bottom border to match the width of the rest we need to override the default 3px bottom border set on `#wikiArticle`. Using ids for styling is bad, but we are in this hot soup already ;p, you can create a higher specificity selector like so:~~

```
#wikiArticle.archive-content {
  border-bottom-width: 20px;
}
```

Reply:
Actually, scratch all of that. Move the `archive-content` class to the `div` with the `id` content, and remove the negative margins for the archive styling.

```
.archive-content {
    background-color: $red-light;
    border: 10px solid $red;
    border-image: repeating-linear-gradient(-45deg, $red 0, $red 5px, $yellow 5px, $yellow 10px, $red 10px) 10 10 repeat;
    padding: $mobile-center-spacing;
}

@media #{$mq-tablet-and-up} {
    .archive-content {
        border-width: 20px;
        border-image: repeating-linear-gradient(-45deg, $red 0, $red 10px, $yellow 10px, $yellow 20px, $red 20px) 20 20 repeat;
        padding: $gutter-width;
    }
}
```